docs/content/preview/secure/enable-authentication/ycql.md

```diff
@@ -137,6 +137,43 @@ You should see the following output.
 (3 rows)
 ```
 
+#### Reset password for a superuser
+
+Superusers can alter the passwords for other users including the admin user. YCQL provides a feature to enable superusers to reset a lost/forgotten password using the[`ycql-allow-non-authenticated-password-reset`](../../../reference/configuration/yb-tserver/#ycql-allow-non-authenticated-password-reset) YB-TServer GFlag.
+Use the [`ycql-allow-non-authenticated-password-reset`](../../../reference/configuration/yb-tserver/#ycql-allow-non-authenticated-password-reset) YB-TServer GFlag in conjunction with the [`use_cassandra_authentication`](../../../reference/configuration/yb-tserver/#use-cassandra-authentication) YB-TServer GFlag to enable the password reset feature.
```

Comment:
Please use underscore (`_`) instead of dash (`-`) as a separator. The flag name is `ycql_allow_non_authenticated_password_reset`.